pybravia/client.py

```diff
@@ -178,6 +179,10 @@ async def send_req(
             if response.status == 200:
                 result = await response.json() if json else True
                 _LOGGER.debug("Response result: %s", result)
+                if isinstance(result, dict) and "not power-on" in result.get(
+                    "error", []
+                ):
+                    raise BraviaTVTurnedOff
```

Comment:
I propose to rename it to a more general:
```suggestion
                    raise BraviaTVUnavailable
```

Reply:
Hmm I think that `BraviaTVUnavailable` will suggest that we should mark the device as unavailable in HA, and we don't want to do that, we just want to set state to `off`. Maybe `BraviaTVIdleMode` or `BraviaTVSleeping` would be better if you don't like `TurnedOff`?

Reply:
Right. Let's leave this name then.